app/googleanalytics/pixels.py

```diff
@@ -1,35 +1,53 @@
-from urllib.parse import urlencode
-
+import os
 from flask import current_app
+from urllib.parse import quote
+
+TEST_DOMAIN = 'https://test-api.va.gov/vanotify/'
+DEV_DOMAIN = 'https://dev-api.va.gov/vanotify/'
+PERF_DOMAIN = 'https://sandbox-api.va.gov/vanotify/'
+STAGING_DOMAIN = 'https://staging-api.va.gov/vanotify/'
+PROD_DOMAIN = 'https://api.va.gov/vanotify/'
+
```

Comment:
I think we can move this to app.config and only have one variable. Then the ENVIRONMENT_DOMAINS can be scrapped. A nit, really.

Reply:
Oh I like this. I'll implement this.